biggus/__init__.py

```diff
@@ -561,6 +563,175 @@ def _normalise_keys(self, keys):
         return keys
 
 
+class ArrayContainer(Array):
```

Comment:
Any array which holds an array ends up implementing all of the array interface as wrapper properties. This prevents that.

NewAxesArray subclasses.